web_generator/bin/update_bindings.dart

```diff
@@ -128,6 +131,25 @@ $_startComment
   }
 }
 
+Future<String> _webPackageLanguageVersion(String pkgPath) async {
```

Comment:
IIUC, you want to format the `web` package using the language version that it is tied to within its pubspec?

Did you end up noticing lots of changes wrt formatting when you ran the generator before this PR?

Reply:
yep! mostly comma removals, but there were some spacing things that also showed up

Reply:
ack, makes sense. thanks!